pkg/server/app/app.go

```diff
@@ -96,8 +97,14 @@ func (ops *AppOperations) Create(user *storage.User, app *App) error {
 		return teresa_errors.NewInternalServerError(err)
 	}
 
+	defer func() {
+		if Err != nil {
```

Comment:
⚠️  Be careful! you need to check if error is different of `ErrAlreadyExists` or `ErrPermissionDenied` before perform the `delete namespace` action